lib/core/presentation/pages/profile/views/tabs/profile_collectible_tab_view.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:app/core/domain/token/token_enums.dart';
+import 'package:app/core/domain/user/entities/user.dart';
+import 'package:app/core/presentation/pages/profile/views/list/profile_nft_collected_list_view.dart';
+import 'package:app/core/presentation/pages/profile/views/list/profile_nft_created_list_view.dart';
+import 'package:app/core/presentation/pages/profile/views/list/profile_nft_on_sale_list_view.dart';
+import 'package:app/core/presentation/pages/profile/views/list/profile_nft_sold_list_view.dart';
+import 'package:app/core/presentation/pages/profile/views/tabs/base_sliver_tab_view.dart';
+import 'package:app/core/presentation/widgets/lemon_chip_widget.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:flutter/material.dart';
+
+class ProfileCollectibleTabView extends StatefulWidget {
+  final User user;
+  const ProfileCollectibleTabView({
+    super.key,
+    required this.user,
+  });
+
+  @override
+  State<ProfileCollectibleTabView> createState() => _ProfileCollectibleTabViewState();
+}
+
+class _ProfileCollectibleTabViewState extends State<ProfileCollectibleTabView> {
+  double get _filterBarHeight => 72;
```

Comment:
Do we need some responsive height value ?

Reply:
<img width="501" alt="Screenshot 2023-07-07 at 08 50 16" src="https://github.com/lemonadesocial/lemonade-flutter/assets/28992487/8037aea4-4323-4537-86f2-fe9fa50edd19">
I think the height of the filter bar wont change through out the device screen, that's why I leave it hardcode